lib/inspector.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
```

Comment:
Should we add docs for this change ?

Reply:
I won't be writing docs until I confirm this feature actually makes sense..
(now I am leaning towards no)